android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UriBeaconDiscoveryService.java

```diff
@@ -186,6 +187,7 @@ private static String generateMockBluetoothAddress(int hashCode) {
   private void initialize() {
     mNotificationManager = NotificationManagerCompat.from(this);
     mMdnsUrlDiscoverer = new MdnsUrlDiscoverer(this, UriBeaconDiscoveryService.this);
+    mSsdpUrlDiscoverer = new SsdpUrlDiscoverer(this,UriBeaconDiscoveryService.this);
```

Comment:
Better formatted with an extra space before UriBeaconDiscoveryService.this:
mSsdpUrlDiscoverer = new SsdpUrlDiscoverer(this, UriBeaconDiscoveryService.this);